crates/turbopack-ecmascript/tests/tree-shaker/analyzer/complex/output.md

```diff
@@ -30,7 +30,8 @@ console.log(dog);
 ```
 
 - Side effects
-- Reads: `console`, `dog`
+- Reads: `dog`
+- Write: `console`
```

Comment:
Technically we don't need to track these "external" variables at all, since they are already lead to `Side effects`